templates/base/footer.tmpl

```diff
@@ -22,7 +22,7 @@
 		<script src='https://hcaptcha.com/1/api.js' async></script>
 	{{end}}
 {{end}}
-	<script src="{{AssetUrlPrefix}}/js/index.js?v={{AssetVersion}}" onerror="alert('Failed to load asset files from ' + this.src + ', please make sure the asset files can be accessed and the ROOT_URL setting in app.ini is correct.')"></script>
```

Comment:
But wasn't this the exact reason why you added this tooltip especially with the `ROOT_URL` in the first part?
Given that is still often enough the reason, we should keep it.

Reply:
No, this is not. See the original PR.

Reply:
> Isn't {{AssetUrlPrefix}} built from ROOT_URL ?

No, not directly.

> Or some other app.ini setting ?

Somewhat, but not directly. See the PR's comment: ex: https://gitea.com/gitea/helm-chart/issues/392

>  It would be useful to be hinted about which app.ini setting affects that url...

There are many different cases. In most cases, users could fix it by themselves. If not, it's better  to collect commono problems and write a FAQ.